lib/fastlane/actions/create_keychain.rb

```diff
@@ -0,0 +1,67 @@
+module Fastlane
+  module Actions
+    class CreateKeychainAction < Action
+      def self.run(params)
+        commands = []
+        commands << sh("security create-keychain -p #{params[:password]} #{params[:name]}")
```

Comment:
What if `params[:password]` contains a space? I think we have to wrap it around `'`, right?